### ISSUE TITLE
请问什么时候可以支持多模态

### ISSUE THREAD

**Issue by q-Mony** [bestony/LarkGPT#1]
请问什么时候可以支持多模态

**Comment by ghernanda** [comment]
Hi Bestony,

Thanks for the code & guide. However, seems like there's in issue. I've followed your guide, but the LarkGPT only returns to "this is too difficult for me, ask my owner"

can advise how to resolve this?

![image](https://github.com/bestony/LarkGPT/assets/157116941/9426009b-bb37-4a17-88af-3d3ddb46f071)

**Comment by bestony** [comment]
> Hi Bestony,
> 
> Thanks for the code & guide. However, seems like there's in issue. I've followed your guide, but the LarkGPT only returns to "this is too difficult for me, ask my owner"
> 
> can advise how to resolve this?
> 
> ![image](https://private-user-images.githubusercontent.com/157116941/298044908-9426009b-bb37-4a17-88af-3d3ddb46f071.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU2NjQ3MjIsIm5iZiI6MTcwNTY2NDQyMiwicGF0aCI6Ii8xNTcxMTY5NDEvMjk4MDQ0OTA4LTk0MjYwMDliLWJiMzctNGExNy04OGFmLTNkM2RkYjQ2ZjA3MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDExOVQxMTQwMjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMzIzMGM2YWZmNmU3ZDYyMTIwN2Q1ZWMwOWY2NDc0NzgxZDQ2NjdhMmY5ZGQ2ZTlkY2Y5YzA3OTA3ZDcyY2IwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.ABw0VZhvb0l_DAqm6E3wPIBpKXdC9onZGT6uJAyUit0)

Hi,you can see Aircode's log to check if someting wrong for you.

**Comment by bestony** [comment]
> 请问什么时候可以支持多模态

最近出了 GPT4V 的接口，应该最近可以支持上。

**Comment by ghernanda** [comment]
this is the error log:

error: {
    message: 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.',
    type: 'insufficient_quota',
    param: null,
    code: 'insufficient_quota'
    
  I do have paid subscription for my openai but should be able to use the GPT 3.5 right?

**Comment by bestony** [comment]
> this is the error log:
> 
> error: { message: 'You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors.', type: 'insufficient_quota', param: null, code: 'insufficient_quota'
> 
> I do have paid subscription for my openai but should be able to use the GPT 3.5 right?

no, you just have paid subscription for chat gpt ,not for OpenAI's API, you need set biiling in platform.openai.com
